README.md

```diff
@@ -134,6 +134,18 @@ pip install paper2remarkable
 
 This installs the ``p2r`` command line program.
 
+## Alfred
+
+Install the [Alfred workflow][workflow].
+
+You can then use `rm` command and `rmb` (for the `--blank` pages to insert blank
+pages between pages for notes) with a URL passed. The global shortcut `Alt-P`
+will send the current selection to `p2r`.
+
+<img width="605" alt="image" src="https://user-images.githubusercontent.com/97400/75118215-311e7300-5646-11ea-92eb-53317ed7f18e.png">
```

Comment:
Could we add the image to this repo? I'm fine having a ``.github`` directory in the root of the repo for this kind of stuff, just so it's all together.